### PR TITLE
[DSP] only use textbox in dialogs and use CVariant

### PR DESCRIPTION
**Pull request by MartijnKaijser**

### CODE REVIEW

File: xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp [hunk 1]
```diff
@@ -764,7 +764,7 @@ bool CActiveAEDSP::UpdateAddons(void)
   {
     m_noAddonWarningDisplayed = true;
     CSettings::Get().SetBool("audiooutput.dspaddonsenabled", false);
-    CGUIDialogOK::ShowAndGetInput(24055, 24056, 24057, 24058);
+    CGUIDialogOK::ShowAndGetInput(24055, 24056, 24057, 24058); // Completely wrong labels
```
Comment (MartijnKaijser) [inline xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp]: @AchimTuran ping on the incorrect labels

**Comment by AchimTuran** [comment]
@AlwinEsch can you help?

**Comment by AlwinEsch** [comment]
Hi @AchimTuran and @MartijnKaijser, is currently on build and I do a check over a day about it.

**Comment by AlwinEsch** [comment]
Has checked, there comes a bit more to do, dsp is currently on startup really unstable and brings a lot of crashes. There was something changed on the startup way inside CApplication and produce this.

About this wrong strings, was thought to bring a error dialog on startup without any dsp add-on installed. Think is not really needed and brings also the problem that it becomes called before the gui is ready. I fix then the other errors and removing this OKDialog call (dsp also disabled from there).

**Comment by AchimTuran** [comment]
@AlwinEsch thanks for your help. You're right the next steps for AudioDSP is to make it more stable and fixing the Add-ons.

**Pull request closed without merging**